docs/setup.asciidoc

```diff
@@ -4,16 +4,43 @@
 Follow these steps to start reporting your Android application's performance to Elastic APM:
 
 1. <<gradle-setup>>.
-2. <<application-setup>>.
+2. <<manual-setup>>.
 3. <<compile-and-run>>.
-4. <<manual-setup>>.
-5. <<whats-next>>.
+4. <<whats-next>>.
 
 [float]
 [[gradle-setup]]
 === Set up Gradle
 
-First, add the https://plugins.gradle.org/plugin/co.elastic.apm.android[Elastic APM agent plugin] to your application's `build.gradle` file as shown below:
+[float]
+[[gradle-requirements]]
+==== Requirements
+
+|===
+|Requirement |Minimum version
+
+|Android Gradle plugin
+|7.4.0
+
+|Android API level
+|24
+
+|===
+
+[float]
+[[minsdk-24-support]]
+==== For projects using minSdk version < 26
```

Comment:
I think this is actually called `minSdkVersion`?

```suggestion
==== For projects using minSdkVersion < 26
```

Reply:
If I'm wrong, ingore my suggestions in the next paragraph.

Reply:
Yeah in the code is called `minSdkVersion` so I think it makes sense to have it like that in here as well. Cheers!